elastic/apm-server
# Add `span.kind` field mapping

## Motivation/summary

Map `span.kind` as a keyword field, so it can be used in filters and aggregations.

## Checklist

~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)~
~- [ ] Documentation has been updated~

## How to test these changes

1. Run apm-server, check legacy index template has `span.kind` mapped as a keyword
2. Install integration, check `traces-apm-*` index template has `span.kind` mapped as a keyword

## Related issues

Closes https://github.com/elastic/apm-server/issues/6512
